SPECS/kernel/CVE-2024-21803.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2024-21803 - fixed in 6.6.X
```

Comment:
I am not able to track down this patch for some reason, the link gives me access denied and have been unable to set up an account. https://bugzilla.openanolis.cn/show_bug.cgi?id=8081

Reply:
I found it through https://ubuntu.com/security/CVE-2024-21803 who determined this was a duplicate of CVE-2023-51779 who has this as a patch https://nvd.nist.gov/vuln/detail/CVE-2023-51779